plugins/googleanalytics/src/org/pentaho/di/trans/steps/googleanalytics/GaInputStep.java

```diff
@@ -26,6 +26,7 @@
 import java.util.List;
 
 import com.google.api.services.analytics.Analytics;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Please remove this import - we don't have a dependency on guava which defines this annotation and we don't want to add it just to make an annotation that can be replaced by a comment for it...